platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -893,7 +890,14 @@ private void loadStateIntoConsensus(@NonNull final SignedState signedState) {
         Objects.requireNonNull(signedState);
 
         consensusRef.get().loadFromSignedState(signedState);
-        shadowGraph.startWithExpiredThreshold(consensusRef.get().getMinGenerationNonAncient());
+
+        final NonAncientEventWindow eventWindow = new NonAncientEventWindow(
```

Comment:
Does this need a conditional flag based on ancient mode?   Or a comment to switch to round based after migration?